virttest/qemu_vm.py

```diff
@@ -2839,7 +2839,7 @@ def _stop_daemons(self):
     def create(self, name=None, params=None, root_dir=None,
                timeout=120, migration_mode=None,
                migration_exec_cmd=None, migration_fd=None,
-               mac_source=None):
+               mac_source=None, safe_destroy=True):
```

Comment:
@pevogam I can see the point of the new argument being introduced to `create`, however I'd prefer another approach, and that is to remove the invoking of `destroy` from `create` in order to make the implementation of `create` more proper. Then each VM launching workflow can fully control the way of if and how `destroy` would be called. Let me know your opinion, thanks!

Reply:
Hi @pevogam 
I maybe prefer @luckyh's opinions, firstly, for adding `safe_destroy` here, I think it is not a better way to understand the definition of method `create`, I mean that the arguments should be related to the creating behaviors. And the job of creating a VM just focuses on creating, not including the destroy VM this step should be decoupled from the `create`.
I am glad to know your opinions., Thanks.

Reply:
I agree with both of you here, was just trying to avoid breaking the behavior for other people since AFAIK `vm.create()` is a ubiquitous line that is used throughout the project, not to mention possible external projects using the `vm` object too. But if you feel bold let's try it - I will push a change here that drops the "re-creation" functionality and update any lines I find at least in Avocado VT.

Reply:
Pushed and CI passed